test/unit/math/prim/mat/prob/normal_test.cpp

```diff
@@ -0,0 +1,42 @@
+#include <stan/math/prim/mat.hpp>
+#include <gtest/gtest.h>
+#include <boost/math/distributions.hpp>
+#include <boost/random/mersenne_twister.hpp>
+#include <test/unit/math/prim/mat/prob/vector_rng_test_helper.hpp>
+
+using Eigen::Dynamic;
+using Eigen::Matrix;
+
+struct dist_wrapper {
```

Comment:
This is a functor that allows me to pass a templated _rng around as a variable. It has to be a functor instead of a function itself for the templating to work (https://stackoverflow.com/a/15651770/3769360).